Targets/STM32F4/STM32F4_UART.cpp

```diff
@@ -27,23 +27,30 @@
 #define STM32F4_UART_DATA_BIT_LENGTH_8    8
 #define STM32F4_UART_DATA_BIT_LENGTH_9    9
 
+//Default buffers size
+#define STM32F4_UART_TX_BUFFER_SIZE 256
```

Comment:
Keep these in Device.h but change the name to `STM32F4_UART_DEFAULT_TX_BUFFER_SIZE` and `STM32F4_UART_DEFAULT_RX_BUFFER_SIZE`. Same for NPX and AT.